Riot/Modules/KeyBackup/Setup/KeyBackupSetupCoordinator.swift

```diff
@@ -44,8 +44,9 @@ final class KeyBackupSetupCoordinator: KeyBackupSetupCoordinatorType {
     func start() {
     
         // Set key backup setup intro as root controller
+        let isABackupAlreadyExists = !(self.session.crypto.backup?.state == MXKeyBackupStateDisabled)
```

Comment:
The case `MXKeyBackupStateCheckingBackUpOnHomeserver` may be boring but we can iterate on that later.
The worse scenario is the app running offline.